DEVELOPMENT.md

```diff
@@ -79,6 +79,41 @@ Verify installed plugins
 tanzu plugin list
 ```
 
+*Publishing Apps CLI Plug-in artifact to Github Container Registry*
```

Comment:
clarification : are we supporting to upload in to any registry or only to github container registry ?

Reply:
Not sure how to do it for other registries. This title is entirely related to the steps needed to authenticate and publish **only** to ghcr. Should we have a more generic section that explains how to do it for any kind of registry?